com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/DebugUtility.java

```diff
@@ -40,10 +40,27 @@
 import com.sun.jdi.request.StepRequest;
 
 public class DebugUtility {
-
-    public static IDebugSession launch(VirtualMachineManager vmManager, String mainClass, String programArguments, String vmArguments, List<String> classPaths,
-            String cwd, String[] envVars) throws IOException, IllegalConnectorArgumentsException, VMStartException {
-        return DebugUtility.launch(vmManager, mainClass, programArguments, vmArguments, String.join(File.pathSeparator, classPaths), cwd, envVars);
+    /**
+     * Launch a debuggee in suspend mode.
```

Comment:
suspend ==> suspended

Reply:
`Suspend Mode` is OK, you can google it, there are many places calling it `Suspend Mode`.